fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/model/Tablespace.java

```diff
@@ -44,6 +44,17 @@ public void apply(IDatabaseAdapter target) {
         }
     }
 
+    @Override
+    public void apply(Integer priorVersion, IDatabaseAdapter target) {
```

Comment:
don't do anything here. we shouldn't make a new tablespace, or alter here.

Reply:
now it throws if you try doing an update:
```
        if (priorVersion != null && priorVersion > 0) {
            throw new UnsupportedOperationException("Modifying tablespaces is not supported");
        }
        apply(target);
```